packages/core-mobile/app/new/routes/_layout.tsx

```diff
@@ -1,23 +1,49 @@
-import { useEffect } from 'react'
+import { useEffect, useMemo } from 'react'
 import 'react-native-reanimated'
 import Bootsplash from 'react-native-bootsplash'
 
 import React from 'react'
-import { K2AlpineThemeProvider } from '@avalabs/k2-alpine'
+import {
+  K2AlpineThemeProvider,
+  lightTheme,
+  darkTheme
```

Comment:
the app should never use these 2 directly. everything should be done through theme.colors.

Reply:
fixed.